docs/report_queries/multiple_logical_definitions.md

```diff
@@ -0,0 +1,18 @@
+# Multiple Logical Definitions
+
+**Problem:** A class has more than one logical definitions. This should be avoided, as logical definitions can interact adversely.
```

Comment:
I don't like "more than one logical definitions", because we allow multiple subClassOf assertions.

What about "A class has more than one equivalent class definition." ?

Reply:
This makes sense, probably we should also rename the files as well 🗡️

Reply:
Name for that check maybe `multiple_equivalent_class_definitions`

Reply:
Renamed.